sccm/apps/deploy-use/copy-applications.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
I suggest not including this new article, as it's too short. Instead, include the intro paragraph (suggested rewording below) and the process as a new H3 section (`###`) in this existing article: [Manage applications](https://docs.microsoft.com/en-us/sccm/apps/deploy-use/management-tasks-applications#manage-applications). That article talks about other actions in a table format. I'm not a fan of tables for this purpose, so if you also want to break up that table so that each row is a separate H3, that's great. 😄 

Suggestion rewording for intro blurb:

> Copy, or duplicate, Configuration Manager applications to create a new one. This action is useful to test something or when you need to create a similar application.

Reply:
Since I had made this change in master before (poor newbie mistake on my part), I reforked the actual current master, branched, and then opened #1584 for this one.